### DISCUSSION TITLE
Is there any way to set the name of the ecr repository that is created when deploying?

### DISCUSSION THREAD

**Discussion by fclemonschool** [serverless/serverless#10883] [Q&A] [answered]
I can't find a way to set the ecr repository name in serverless.yml.

Is there any way to set the name of the ecr repository that is created when deploying?

**Comment by pgrzesik** [comment, marked as answer]
Hey, there's no way to override the name of the repository - it's always created following the pattern: `serverless-${serviceName}-${stage}`

**Reply by fclemonschool** [reply]
It's sad that that feature doesn't exist. 
It would be nice to be able to customize the name in the future.

**Reply by safuente** [reply]
Yeah it would be specially useful when you already have the repository name because you are deploying manually and you want to migrate the deployment to serverless option
